sdk/python/pvsite_datamodel/sqlmodels.py

```diff
@@ -20,7 +18,7 @@
 class CreatedMixin:
     """Mixin to add created datetime to model."""
 
-    created_utc = Column(DateTime(timezone=True), default=lambda: datetime.utcnow())
```

Comment:
I removed the timezone information for simplicity and consistency with the other DateTime fields.